docs/chaos/index.md

```diff
@@ -30,20 +30,30 @@ builder
 // Finally, configure chaos strategies if you want to inject chaos.
 // These should come after the regular resilience strategies.
 
-// 2% of invocations will be injected with chaos
-const double InjectionRate = 0.02;
+// 2% of total requests will be injected with chaos fault.
+const double faultInjectionRate = 0.02;
+// For the other 98% of total requests, 50% of them will be injected with latency. Then 49% of total request will be injected with chaos latency.
+// Latency injection does not return early.
+const double latencyInjectionRate = 0.50;
+// For the other 98% of total requests, 10% of them will be injected with outcome. Then 9.8% of total request will be injected with chaos outcome.
+const double outcomeInjectionRate = 0.10;
+// For the other 89.2% of total requests, 1% of them will be injected with behavior. Then 0.892% of total request will be injected with chaos behavior.
+const double behaviorInjectionRate = 0.01;
```

Comment:
Use "all" or "the total" here.

It might also read better to say "the remaining" rather than "the other".